### PR TITLE
Upgrade packages and cov cli

### LINKED ISSUE #11: Update ws 1.1.0 dependency to fix vulnerability
#### Bug Report
Update ws 1.1.0 dependency to fix vulnerability

##### Screenshots or link to CodePen/Plunker/JSfiddle

error

### PR DESCRIPTION
**Pull request by christianmemije**
## Description

- Ran `yarn audit` and upgraded all packages that had high risk vulnerabilities. 
- Upgraded cov cli to work with ng cli 7
- Fixes #11 

#### Test Steps
- [ ] `yarn`
- [ ] Follow steps in covalent-e2e-coverage/README.md

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:release` still works.

### CODE REVIEW

File: docs/CLI.md [hunk 1]
```diff
@@ -52,64 +54,59 @@ cov versions
...
 * Starts the ng server with flags needed to run e2e coverage
-* Same as `node --max_old_space_size=5048 ./node_modules/.bin/ng serve --aot --prod --sourcemap=true --build-optimizer --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true --aot --prod --optimization`
 
 ```
 cov e2e
 ```
 * Runs the end to end tests against local code base
-* Same as `ng e2e --config --proxy-config e2e/proxy.conf.json`
+* Same as `ng e2e`
```
Comment (jeremysmartt) [inline docs/CLI.md]: don't we still need the proxy conf on here?

File: docs/CLI.md [hunk 2]
```diff
@@ -52,64 +54,59 @@ cov versions
...
-* Same as `ng serve --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true`
 
 ```
-cov serve-e2e
+cov serve-prod
 ```
 * Starts the ng server with flags needed to run e2e coverage
-* Same as `node --max_old_space_size=5048 ./node_modules/.bin/ng serve --aot --prod --sourcemap=true --build-optimizer --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true --aot --prod --optimization`
```
Comment (jeremysmartt) [inline docs/CLI.md]: why did you remove the memory from this documentation?

File: docs/CLI.md [hunk 3]
```diff
@@ -52,64 +54,59 @@ cov versions
...
 * Runs the end to end tests against local code base
-* Same as `ng e2e --config --proxy-config e2e/proxy.conf.json`
+* Same as `ng e2e`
 
 ```
 cov e2e-for-coverage
 ```
 * Runs the end to end tests against the e2e proxy
-* Same as `ng e2e -pc proxy.conf.json --no-serve --base-href=http://localhost:5050`
+* Same as `ng e2e --baseUrl=http://localhost:5050 --devServerTarget=`
```
Comment (jeremysmartt) [inline docs/CLI.md]: don't we still need the proxy conf on here?

File: docs/CLI.md [hunk 4]
```diff
@@ -52,64 +54,59 @@ cov versions
...
 ```
 * Runs the unit tests
-* Same as `ng test --code-coverage --single-run --sourcemap=false`
+* Same as `ng test --sourceMap=false --code-coverage --watch=false`
 
 ```
 cov build
 ```
 * Compiles the code with needed flags
-* Same as `node --max_old_space_size=8192 ./node_modules/.bin/ng build --aot --prod --sourcemap=false`
```
Comment (jeremysmartt) [inline docs/CLI.md:85]: why did you remove the memory from this documentation?

**Review by jeremysmartt — CHANGES_REQUESTED** [review]

File: docs/CLI.md (hunk 1 above)
Comment (christianmemije) [reply docs/CLI.md]: I believe the ng cli looks at the angular.json and finds the proxy config relative to the `sourceRoot`, which is the `e2e` dir.

File: docs/CLI.md (hunk 3 above)
Comment (christianmemije) [reply docs/CLI.md]: Same as above.

File: docs/CLI.md (hunk 2 above)
Comment (christianmemije) [reply docs/CLI.md]: Oops.

File: docs/CLI.md (hunk 4 above)
Comment (christianmemije) [reply docs/CLI.md:85]: Oops.

**Review by jeremysmartt — APPROVED** [review]
